db/migrate/20231031175222_create_users.rb

```diff
@@ -5,7 +5,7 @@ def change
       t.string :email
       t.integer :role
       t.string :movie_history, default: ""
-      t.string :password
+      t.string :password_digest
```

Comment:
That makes sense!

Reply:
Again, this change was already made on our side, hopefully no merge conflict